### ISSUE TITLE
indentation of closing paren for multiline functions

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1250]
```diff
  MergeJSFiles(new string[] {
                 GetDecompressor(),
                 Paths.Combine(buildToolsDir, "UnityConfig"),
                 Paths.Combine(args.stagingAreaData, kOutputFileLoaderFileName),
             }, unityLoader
-  );
+                );
```

```diff
  throw new System.Exception(
                     "'Fast Rebuild' option requires prebuilt JavaScript version of Unity engine. The following files are missing: "
                     + (!File.Exists (UnityNativeJs) ? "\n" + UnityNativeJs : "")
                     + (!File.Exists (UnityNativeJs + ".mem") ? "\n" + UnityNativeJs + ".mem" : "")
-                );
+                    );
```

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60001-UNI-2650.cpp" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60001-UNI-2650.cpp"
index b9ced77..273e4dc 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60001-UNI-2650.cpp"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60001-UNI-2650.cpp"
@@ -1,13 +1,13 @@
 MergeJSFiles(new string[] {
     GetDecompressor(),
     Paths.Combine(buildToolsDir, "UnityConfig"),
     Paths.Combine(args.stagingAreaData, kOutputFileLoaderFileName),
 }, unityLoader
-);
+    );
 
 
 throw new System.Exception(
     "'Fast Rebuild' option requires prebuilt JavaScript version of Unity engine. The following files are missing: "
     + (!File.Exists(UnityNativeJs) ? "\n" + UnityNativeJs : "")
     + (!File.Exists(UnityNativeJs + ".mem") ? "\n" + UnityNativeJs + ".mem" : "")
-);
+    );
```

Debug support files:
---
[60001-UNI-2650.cpp](https://gist.githubusercontent.com/mihaipopescu/fa68023b5162aae09a29059e7d53fa8f/raw/b9ced7737b3d6f402c1879d758763b40e9b953e5/60001-UNI-2650.cpp)
[60001-UNI-2650.cpp.log](https://gist.githubusercontent.com/mihaipopescu/fa68023b5162aae09a29059e7d53fa8f/raw/3a592a294c437acd6bd13ce7b3e2876e12177c4b/60001-UNI-2650.cpp.log)
[60001-UNI-2650.cpp.unc](https://gist.githubusercontent.com/mihaipopescu/fa68023b5162aae09a29059e7d53fa8f/raw/1a648806626aa62ec7f4060535a9e68d87da984e/60001-UNI-2650.cpp.unc)
[Uncrustify.Cpp.cfg](https://gist.githubusercontent.com/mihaipopescu/fa68023b5162aae09a29059e7d53fa8f/raw/23d62d9dba5caf352fdc1793792e98eed4b81929/Uncrustify.Cpp.cfg)
[UNI-2650.cpp](https://gist.githubusercontent.com/mihaipopescu/fa68023b5162aae09a29059e7d53fa8f/raw/b9ced7737b3d6f402c1879d758763b40e9b953e5/UNI-2650.cpp)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by guy-maurel** [comment]
You need to use the option `indent_paren_close=2`

**Comment by gmaurel** [comment]
Fixed at 3c50d0a29b551320a69a43d98dcd5d0e2df1812f, Dec 13 15:24:21 2017
